### ISSUE TITLE
How to gradually change a variable's value?

### ISSUE THREAD

**Issue by jiumem** [dmlc/mxnet#3585]
See a multi task loss: toatal_loss = loss1 + lambda*loss2. How can I gradually change the lambda's value?
I know for tensorflow :
```python
lambda = tf.Variable(0.1, trainable=False)
lambda_larger = tf.assign(lambda, lambda*2)
# some train code here
...
if (iter+1) % 1000:
    sess.run(lambda_larger)
```
How could mxnet do this?

**Comment by piiswrong** [comment]
you can make a an input variable and feed it through your data iter

**Comment by VoVAllen** [comment]
lambda=mx.symbol.BlockGrad(mx.symbol.Variable(name='lambda'))
lambda=lambda*lambda (maybe this need element wise symbol?)

**Comment by jiumem** [comment]
Oh，thanks！
